src/main/kotlin/org/dif/crypto/key/RecipientKeySelector.kt

```diff
@@ -13,28 +13,28 @@ import org.dif.utils.isDIDFragment
 class RecipientKeySelector(private val didDocResolver: DIDDocResolver, private val secretResolver: SecretResolver) {
     fun findVerificationKey(signFrom: String): Key = Key.wrapVerificationMethod(
         let {
-            check(isDIDFragment(signFrom)) { "'DID URL' was excepted" }
+            check(isDIDFragment(signFrom)) { "'DID URL' is expected as a signature verification key. Got: $signFrom" }
 
             val (did) = divideDIDFragment(signFrom)
             didDocResolver.resolve(did)
                 .map { it.findVerificationMethod(signFrom) }
-                .orElseThrow { throw DIDDocNotResolvedException(did) }
+                .orElseThrow { throw DIDUrlNotFoundException(did) }
```

Comment:
It looks like `DIDUrlNotFoundException` will be thrown when `didDocResolver.resolve(did)` returns null, won't it? I think we should check this two cases separately (did doc resolving and finding elements within successfully resolved did doc).
It's OK to fix it later in the PR devoted to exception handling.